src/helpers/action.ts

```diff
@@ -34,7 +34,7 @@ export function debugInfoForReference(reference: any): string {
   let parent;
   let property;
 
-  if (reference == null) { return message; }
```

Comment:
Hmm.  I think this `==` may have been intentional (as it matches `null` and `undefined` but not other falsey values).  I think this should be updated to `reference === null || reference === undefined`.